### ISSUE TITLE
[Insight] Logical operators should be avoided - in _settings.php, line 203/205

### ISSUE THREAD

**Issue by derchrisuk** [webSPELL/webSPELL-4.2.3#126]
**[in _settings.php, line 203](https://github.com/webSPELL/webSPELL-4.2.3/blob/4cd4a505c1bb38505233f8bcce37fed57bf0e474/_settings.php#L203)**

> The `or` operator does not have the same precedence as `||`.
            This could lead to unexpected behavior, use `||` instead.

```php
        $_mysql_querys[ ] = $query;
        if (empty($query)) {
            return false;
        }
        if (DEBUG == "OFF") {
            $result = $_database->query($query) or system_error('Query failed!');
        } else {
            $result = $_database->query($query) or
            system_error(
                'Query failed: ' . '<ul>' .
                '<li>errorno=' . $_database->errno . '</li>' .
```

---------------------

_Posted from [SensioLabsInsight](https://insight.sensiolabs.com/projects/ecedfa24-b238-45fb-9e5e-47850bfb499d/analyses/34?status=violations)_

**Comment by Pascalmh** [comment]
Seems like this was missed here: https://github.com/webSPELL/webSPELL-4.2.3/pull/117

**Comment by derchrisuk** [comment]
Fixed in 32c8d3bd31d7a39a6ae732d5a998fe77394d0f57
